### ISSUE TITLE
Regarding about the output of LSTM_predict.py.

### ISSUE THREAD

**Issue by guowb9** [weir12/DENA#17]
Hi,
I have used the DENA tools to detect RNA m6A in my data, and the "-d" parameter was added when I ran LSTM_predict.py. I got result ${prefix_outfile}_details.tsv like this:

ENST00000000233.9       317     AAACA
df2e44fd-337f-4d04-bcd6-95d9ce62e6ef    1.4540097
21a9b4c5-8ab5-42c2-a561-4aa9fffc6688    -2.8870084
dd4e2a5d-dfbe-45fd-bd18-c4b15ae12bae    -0.56397027
a68bbb3e-93c4-4162-beeb-e1620273977f    0.92993855
6be8d8b0-b897-47ba-a86d-ff83b3e02ea4    0.026935648

As your description: the first column was the read ID; Second column was the m6A-modified probability of this read at the candidate coordinate on ENST00000000233.9). But I find the m6A-modified probability <0 and even >1, is the output result right? I didn't meet  errors when run the process.
Thanks!

**Comment by weir12** [comment]
Hi,
Please check whether the model file name extension is **.dat**

The .dat file triggers the softmax function of pytorch in the if branch. (This step is not required for other machine learning models that we have been tried in the past.)

https://github.com/weir12/DENA/blob/2c584c9a22f2903a1c44abe9a734fef5e9d158c8/step4_predict/LSTM_predict.py#L97

Best wishes
